src/linter/html/linter.ts

```diff
@@ -1,32 +1,27 @@
-import {taskStart} from "../../detectors/util/perf.js";
-import {extractJSScriptTags} from "../../detectors/transpilers/html/parser.js";
-import {LintMessageSeverity} from "../../detectors/AbstractDetector.js";
-import HtmlReporter from "./HtmlReporter.js";
+import {LinterParameters} from "../LinterContext.js";
+import transpileHtml from "./transpiler.js";
+import {Resource} from "@ui5/fs";
 
-import type {TranspileResult} from "../../detectors/transpilers/AbstractTranspiler.js";
-import type {ReadStream} from "node:fs";
+export default async function lintHtml({workspace, context}: LinterParameters) {
+	let htmlResources: Resource[];
+	const filePaths = context.getFilePaths();
+	if (filePaths?.length) {
+		htmlResources = [];
+		await Promise.all(filePaths.map(async (filePath) => {
+			if (!filePath.endsWith(".html")) {
+				return;
+			}
+			const resource = await workspace.byPath(filePath);
```

Comment:
Why not just use `workspace.byGlob()` with `.html` search.
Is it because of `--file-paths` argument?